contrib/crio-wipe/lib.bash

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+set -eu
+
+get_major() {
+    echo "$@" | grep -Po '^(\"|crio version )?\K\d(?=\.\d+\..*)'
+}
+
+get_minor() {
+    echo "$@" | grep -Po '^(\"|crio version )?\d\.\K\d+(?=\..*)'
+}
+
+perform_wipe() {
+    if [[ $WIPE -eq 0 ]]; then
+        echo "Wiping storage"
+        rm -rf $CONTAINERS_STORAGE_DIR
```

Comment:
Consider what if `$CONTAINERS_STORAGE_DIR` is `bad bad news / oh no /path/to/important stuff` :smile: 

Should probably quote `$CONTAINERS_STORAGE_DIR`

Reply:
:O good catch